gateway-internal/deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gateway-internal-deployment
+  labels:
+    app: gateway-internal
+    hail.is/sha: "{{ code.sha }}"
+spec:
+  selector:
+    matchLabels:
+      app: gateway-internal
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: gateway-internal
+        hail.is/sha: "{{ code.sha }}"
+    spec:
+      serviceAccountName: gateway-internal
```

Comment:
Does this service account exist?

Reply:
No. I'm not sure where the gateway service account got defined but it does exist...

Reply:
It looks like the gateway service account has a secret in its configuration. Can we discuss what the role of the service accounts are so I can make the proper one (or delete) for the internal gateway?